typescript/web/src/hooks/use-user.tsx

```diff
@@ -52,10 +52,14 @@ const useUserQuery = <TTypes extends UserTupleTypes>(
 ): TTypes[2] | undefined => {
   const id = useUserId() ?? "";
   const skip = isEmpty(id);
-  const { data } = useQuery<TTypes[0], TTypes[1]>(query, {
+  const { data, error } = useQuery<TTypes[0], TTypes[1]>(query, {
     variables: { id },
     skip,
   });
+
+  if (error) {
```

Comment:
The handling should be more selective and keep the other errors follow the usual procedure.